x/wasm/genesis_test.go

```diff
@@ -82,7 +82,8 @@ func TestInitGenesis(t *testing.T) {
 	q2 := newData.grpcQueryRouter
 
 	// initialize new app with genstate
-	InitGenesis(newData.ctx, &newData.keeper, *genState)
+	_, err = InitGenesis(newData.ctx, &newData.keeper, *genState)
+	require.NoError(t, err)
```

Comment:
🤔 This is failing in this branch.

Reply:
aha... yeah... so, I had... oddities here.

Reply:
I found the root cause for the issue in the fuzzer setup. I will do a PR. btw, this is not the line failing

Reply:
I'm not surprised, and I wonder if it is maybe related to failures at line 127 in the same file.

Reply:
... they were both intermittent.